src/aac/templates/genplug/README.md.jinja2

```diff
@@ -0,0 +1,34 @@
+# {{plugin.name}}
+
+## Plugin Commands
+
+### Shell Command
+
```

Comment:
Add some context here about the shell command with the plugin name -- that's how you'd invoke the plugin. For instance, add some info about how to run the plugin with the help command. `aac {{plugin.name}} -h`